src/common/UserDefaults.cpp

```diff
@@ -71,9 +71,6 @@ void initMaps()
             case UseODDMTS:
                 r = "useODDMTS";
                 break;
-            case ActivateExtraOutputs:
```

Comment:
Removing this means people who have it set in their XML will get a popup; we should leave it in and ignore it with a comment.